### DISCUSSION TITLE
How to disable prev next link in markdown for specific pages?

### DISCUSSION THREAD

**Discussion by ljain112** [vuejs/vitepress#1745] [Q&A]
How to disable prev next link in markdown for specific pages?

prev: false
next: false
Not working in vitepress

**Comment by peterfox** [comment]
You'll need to customise the default theme, as far as I can tell. I copied the default theme into my project and removed the links entirely, as they didn't display the correct ones. Eventually, I'll customise them in my project, but for now, the default theme only allows you to customise the text, not where the link goes or when it appears.

**Comment by MarkusKeck** [comment]
PR https://github.com/vuejs/vitepress/pull/1972
